tests/images.rs

```diff
@@ -322,3 +329,45 @@ pub fn free_local_port() -> Option<u16> {
         .map(|addr| addr.port())
         .ok()
 }
+
+#[test]
+fn zookeeper_check_connection() {
+    struct LoggingWatcher;
+    impl Watcher for LoggingWatcher {
+        fn handle(&self, e: WatchedEvent) {
+            info!("{:?}", e)
+        }
+    }
+    use std::env;
+    use std::time::Duration;
+    let _ = pretty_env_logger::try_init();
+    let free_local_port = free_local_port().unwrap();
+    let docker = clients::Cli::default();
+    let image = images::zookeeper::Zookeeper::default().with_mapped_port((free_local_port, 2181));
+    let node = docker.run(image);
+    let host_port = node.get_host_port(2181).unwrap();
+    fn zk_server_urls(port: u16) -> String {
+        let key = "ZOOKEEPER_SERVERS";
+        match env::var(key) {
+            Ok(val) => val,
+            Err(_) => format!("localhost:{}", port),
+        }
+    }
+    let zk_urls = zk_server_urls(host_port);
+    println!("connecting to {}", zk_urls);
+
+    let zk = ZooKeeper::connect(&*zk_urls, Duration::from_secs(15), LoggingWatcher).unwrap();
+    zk.add_listener(|zk_state| println!("New ZkState is {:?}", zk_state));
+
+    let auth = zk.add_auth("digest", vec![1, 2, 3, 4]);
+    println!("authenticated -> {:?}", auth);
```

Comment:
Are these printlns really necessary? I would prefer to keep the tests free of such output!